CHANGELOG.md

```diff
@@ -8,6 +8,14 @@ All versions prior to 0.9.0 are untracked.
 
 ## [Unreleased]
 
+### Changed
+
+* CLI: The `sigstore verify` command now outputs the inner in-toto statement
+  when verifying DSSE envelopes. If verification is successful, the output
+  will be "OK: $FILENAME" followed by the inner in-toto statement. This allows
```

Comment:
Nit: I think we can remove this part, since `OK:` will be on stderr now.

Reply:
fixed